test/dummy/app/models/article.rb

```diff
@@ -1,5 +1,5 @@
 class Article < ActiveRecord::Base
-  belongs_to :admin_user
+  belongs_to :admin_user, optional: true
```

Comment:
The changes to the model files is required because the belongs_to relationships became required by default in rails 6 (I think), but we were not affected by that change as we ran the old default settings

These are related in that they are required for the app to keep working as is under the new defaults from rails 6.